sklearn/linear_model/tests/test_ridge.py

```diff
@@ -153,6 +156,8 @@ def test_ridge_regression_convergence_fail():
 
 def test_ridge_sample_weights():
     # TODO: loop over sparse data as well
+    # Note: parametrizing this test with pytest results in failed
```

Comment:
Weird, maybe you can create an issue about this, if you haven't already.

Reply:
Opened https://github.com/scikit-learn/scikit-learn/issues/11200